terraform/modules/ecs-user/vpce.tf

```diff
@@ -0,0 +1,26 @@
+# Permits connection from analytical-env to AP frontend
+resource "aws_vpc_endpoint" "ap_frontend_vpce" {
+  vpc_id              = var.vpc.id
+  service_name        = var.ap_frontend_vpce
+  private_dns_enabled = true
+  security_group_ids  = [aws_security_group.ap_frontend_vpce_sg.id]
+  subnet_ids          = var.vpc.aws_subnets_private[*].id
+  vpc_endpoint_type   = "Interface"
+}
+
+resource "aws_route53_zone" "ap_zone" {
```

Comment:
Why are we creating a route53 entry for AP? Shouldn't AP have setup the address in route53, it's an external environment to the Orchestration Service.